src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -1100,7 +1100,7 @@ public Result sendMessage(@ApiParam(value = "Message through Data Channel which
 	@Produces(MediaType.APPLICATION_JSON)
 	public RootRestService.RoomInfo getRoomInfo(@ApiParam(value="Room id", required=true) @PathParam("room_id") String roomId,
 												@ApiParam(value="If Stream Id is entered, that stream id will be isolated from the result",required = false) @QueryParam("streamId") String streamId){
-		return new RootRestService.RoomInfo(roomId,RestServiceBase.getRoomInfoFromConference(roomId,streamId,getDataStore()));
+		return new RootRestService.RoomInfo(roomId,RestServiceBase.getRoomInfoFromConference(roomId,streamId,getDataStore()),RestServiceBase.getEndDate(roomId,streamId,getDataStore()), RestServiceBase.getStartDate(roomId,streamId,getDataStore()));
```

Comment:
I don't think it's a good idea. You make 3 queries to the database in this line. Run one query and collect all results.